### DISCUSSION TITLE
Provider reliability report

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#896] [General] [answered]
I went through the git log of the results file in [ani-cli-status](https://github.com/CoolnsX/ani-cli-status) and analyzed reliability of our current providers in the past month.

I define an outage as either "no embed link" or "no link"
A full outage affects all 3 providers at once

number of commits: 594
number of commits to results file: 99
full outages: 16 (2.7%)
gogoplay: +0 (+0.0%)
animixplay: +5 (+0.8%)
xstreamcdn: +12 (+2.0%)

gogoplay has flawless reliability.
animixplay is only slightly less reliable than gogoplay.
xstreamcdn is vastly less reliable and also fluctuates in terms of returned link count.

**Comment by CoolnsX** [comment, marked as answer]
Thanks for reminding me to update the gogo url in ani-cli-status
